client/log_verifier.go

```diff
@@ -64,9 +64,8 @@ func NewLogVerifierFromTree(config *trillian.Tree) (*LogVerifier, error) {
 		return nil, fmt.Errorf("client: NewLogVerifierFromTree(): TreeType: %v, want %v or %v", got, log, pLog)
 	}
 
-	logHasher, err := registry.NewLogHasher(config.HashStrategy)
-	if err != nil {
-		return nil, fmt.Errorf("client: NewLogVerifierFromTree(): NewLogHasher(): %v", err)
+	if config.HashStrategy != trillian.HashStrategy_RFC6962_SHA256 {
+		return nil, fmt.Errorf("client: NewLogVerifierFromTree(): unknown hasher: %s", config.HashStrategy)
```

Comment:
```golang
if s := config.HashStrategy; s != trillian.HashStrategy_RFC6962_SHA256 {
	return nil, fmt.Errorf("client: NewLogVerifierFromTree(): unknown hasher: %v", s)
```
?
Same below.

Reply:
Done I also made the error messages a bit more consistent!